src/StateNode.ts

```diff
@@ -908,7 +909,7 @@ class StateNode<
           selectedTransition! && state.value // targetless transition
             ? new StateTree(this, path(this.path)(state.value)).absolute
             : undefined,
-        transitions: [selectedTransition],
+        transitions: selectedTransition ? [selectedTransition] : [],
```

Comment:
is this `transitions` field supposed to be used anywhere? I've removed it entirely locally and all tests pass, so maybe we could remove it altogether?

Reply:
It will be; the idea is to eventually remove `tree`, and use `transitions`. This will also be useful for debugging.